rest/blip_sync.go

```diff
@@ -538,14 +538,44 @@ func (bh *blipHandler) handleProposedChanges(rq *blip.Message) error {
 
 //////// DOCUMENTS:
 
-// Pushes a revision body to the client
-func (bh *blipHandler) sendRevision(sender *blip.Sender, seq db.SequenceID, docID string, revID string, knownRevs map[string]bool, maxHistory int) {
-	bh.Logf(base.LevelDebug, base.KeySync, "Sending rev %q %s based on %d known.  User:%s", base.UD(docID), revID, len(knownRevs), base.UD(bh.effectiveUsername))
+func (bh *blipHandler) sendRevOrNorev(sender *blip.Sender, seq db.SequenceID, docID string, revID string, knownRevs map[string]bool, maxHistory int) {
+
 	body, err := bh.db.GetRev(docID, revID, true, nil)
 	if err != nil {
-		base.Warnf(base.KeyAll, "[%s] blipHandler can't get doc %q/%s: %v", bh.blipContext.ID, base.UD(docID), revID, err)
-		return
+		bh.sendNoRev(err, sender, seq, docID, revID, knownRevs, maxHistory)
+	} else {
+		bh.sendRevision(body, sender, seq, docID, revID, knownRevs, maxHistory)
 	}
+}
+
+func (bh *blipHandler) sendNoRev(err error, sender *blip.Sender, seq db.SequenceID, docID string, revID string, knownRevs map[string]bool, maxHistory int) {
```

Comment:
knownRevs and maxHistory aren't relevant for sendNoRev, can be removed.

Reply:
Fixed